### PR TITLE
[WIP] Native VPX decoding in HTML5

### PR DESCRIPTION
**Pull request by TijZwa**
This commit is a follow-up on PR https://github.com/Xpra-org/xpra-html5/pull/198.
It adds native (WebCodecs VideoDecoder) decoding of VP8 and VP9.

The PR needs work and a precise testing.

### CODE REVIEW

File: html5/connect.html [hunk 1]
```diff
@@ -22,7 +22,6 @@
     <link rel="icon" type="image/png" href="favicon.png" />
 
     <script type="text/javascript" src="js/RgbHelpers.js"></script>
-    <script type="text/javascript" src="js/ImageDecoder.js"></script>
```
Comment (totaam) [inline html5/connect.html:25]: Was this meant to be part of this commit?

File: html5/js/Client.js [hunk 2]
```diff
@@ -79,7 +79,7 @@ class XpraClient {
       "rgb32",
       "rgb24",
       "scroll",
-      "void",
+      "void"
```
Comment (totaam) [inline html5/js/Client.js:82]: Is it because the linter is complaining? If so, can we silence it?
(it's much better ***with*** the trailing comma because then you can add to the list without causing a diff on an unrelated line above)

File: html5/js/Client.js [hunk 3]
```diff
@@ -94,7 +94,7 @@ class XpraClient {
       "scroll",
       "webp",
       "void",
-      "avif",
+      "avif"
```
Comment (totaam) [inline html5/js/Client.js:97]: same

File: html5/js/Client.js [hunk 4]
```diff
@@ -1703,7 +1707,11 @@ class XpraClient {
   }
 
   on_mousemove(e, window) {
-    if (this.server_readonly || this.mouse_grabbed || !this.connected) {
+    if (this.mouse_grabbed) {
+      return true;
+    }
+
+    if (this.server_readonly || !this.connected) {
```
Comment (totaam) [inline html5/js/Client.js:1714]: This looks like an unrelated fix. Do you need to rebase?

File: html5/js/ImageDecoder.js [hunk 5]
```diff
@@ -7,62 +7,37 @@
...
+        resizeQuality: "medium"
+      };
+
+      const blob = new Blob([packet[7].buffer], { type: `image/${paint_coding}` });
+      const bitmap = await createImageBitmap(blob, bitmap_options);
+      packet[6] = `bitmap:${coding}`;
+      packet[7] = bitmap;
     }
+    return packet;
   };
```
Comment (totaam) [inline html5/js/ImageDecoder.js:42]: This is the same stuff as #198 - so I am skipping this.

**Review by totaam — COMMENTED** [review]
This one looks easier to merge, after #198

File: html5/connect.html (hunk 1 above)
Comment (TijZwa) [reply html5/connect.html:25]: Yes. ImageDecoder is now using createImageBitmap which we can check directly. There is no need for a check-if-we-can-use-ImageDecoder routine anymore. Since the file is called from a worker, we dont need this import anymore.

File: html5/js/Client.js (hunk 2 above)
Comment (TijZwa) [reply html5/js/Client.js]: force of habit :) fixed.
